assets/airports/kjfk.json

```diff
@@ -31,217 +31,247 @@
             "ceiling": 170,
             "airspace_class": "B",
             "poly": [
-                [40.03355278, -73.71678889],
-                [40.06298611, -73.67481944],
-                [40.09182222, -73.63765278],
-                [40.21133583, -73.46443750],
-                [40.23725111, -73.43697444],
-                [40.40881250, -73.25295639],
-                [40.53523417, -73.08381694],
-                [40.55040944, -73.06578500],
-                [40.62937861, -72.96573806],
-                [40.66119667, -72.87438139],
-                [40.66933944, -72.85094444],
-                [40.70008417, -72.76214944],
-                [40.85703833, -72.76219444],
-                [40.90727111, -72.81511500],
-                [40.92884972, -72.93124639],
-                [40.90660806, -72.96964194],
-                [40.90394167, -72.97413722],
-                [40.90205944, -73.02891694],
-                [40.92364778, -73.13569528],
-                [40.86630556, -73.25213333],
-                [40.88573833, -73.28680056],
-                [40.91201944, -73.33351528],
-                [40.91872542, -73.46516244],
-                [40.87442861, -73.58006139],
-                [40.83117750, -73.69138556],
-                [40.83467944, -73.76042028],
-                [40.99880583, -74.10681333],
-                [41.00331278, -74.15769000],
-                [40.73953111, -74.26430806],
-                [40.66221500, -74.10978583],
-                [40.43231611, -74.08154222],
-                [40.42488611, -74.08773472],
-                [40.38402167, -74.18735028],
-                [40.30673139, -74.26767611],
-                [40.24996028, -74.33294528],
-                [40.15420583, -73.99997306],
-                [40.13753194, -73.89581917],
-                [40.06392917, -73.79860389]
+                ["N40.03355", "W73.71679"],
+                ["N40.06299", "W73.67482"],
+                ["N40.09182", "W73.63765"],
+                ["N40.21134", "W73.46444"],
+                ["N40.23725", "W73.43697"],
+                ["N40.40881", "W73.25296"],
```

Comment:
I don't really see a benefit to this change-- it trades away 3 digits of precision to add two `"` and a `N`. Using straight up decimal degrees like `40.73953111, -74.26430806` is the easiest of all accepted forms for the sim to accept coordinate data, so we shouldn't discourage the use of that form.

Please revert changes to this section.

Reply:
I do see these changes to other sections too-- any chance this is from the qgsopenscope plugin's fixes feature?

Reply:
Of course, with 5 digits after the decimal, it looks like the highest possible error of that rounding would be maybe about a meter off (negligible for anything other than airport layout markings). Is this being done because it's your intent to move all the airports to a consistent coordinate format?

Reply:
It is indeed the output from qgsopenscope. I'll change it back in a sec. @oobayly maybe you could change the coordinate format used in the plugin?

Reply:
@erikquinn I never thought to ask this, is there a particular format that would be most efficient for performance or other reasons that we should be trying to stick to?

Reply:
straight numbers, `[34.612813545, -83.341892057], though really it's negligible since it converts them when importing the data from the file. It's not like it causes a calculation every frame, so it doesn't make an appreciable difference what format the coordinates are given in.